requirement-checker/src/Printer.php

```diff
@@ -148,6 +148,14 @@ public function printv(string $message, int $verbosity, ?string $style = null):
             $this->supportColors ? $this->styles['reset'] : ''
         );
 
-        echo $message;
+        if (getenv('BOX_REQUIREMENTS_CHECKER_LOG_TO_STDOUT') === '1')
+        {
+            // use echo/print to support output buffering
+            print $message;
```

Comment:
why `print` vs `echo`?

Reply:
Just a habit - will not make any difference but updated it to the previous style.